### PR TITLE
[xds] Apply the slowdown factor only once to LRS load reporting period

### PR DESCRIPTION
**Pull request by yousukseung**

### CODE REVIEW

File: test/cpp/end2end/xds/xds_server.h
Comment (drfloob) [inline test/cpp/end2end/xds/xds_server.h:762]: Previously, it was `20 * grpc_test_slowdown_factor() * grpc_test_slowdown_factor()`, and now it's just `20`. Did you mean to leave one of those in?

Comment (yousukseung) [reply test/cpp/end2end/xds/xds_server.h:762]: No, I think 20 seconds will do everywhere and we can make debug build tests shorter. All tests start blocking until the first event occurs, and when I checked this happens in <<100ms so I doubt 20s will cause flakiness.

@markdroth is certainly the better person to make the call so I'm fine to wait for his approval.

Comment (yousukseung) [reply test/cpp/end2end/xds/xds_server.h]: Updated to apply only once to unblock for now. PTAL.

**Review by ctiller — APPROVED** [review]

**Pull request merged**
